src/modules/pair/services/pair.compute.service.ts

```diff
@@ -363,6 +387,18 @@ export class PairComputeService implements IPairComputeService {
                 this.computeSecondTokenLockedValueUSD(pairAddress),
             ]);
 
+        if (
+            new BigNumber(firstTokenLockedValueUSD)
+                .minus(secondTokenLockedValueUSD)
+                .abs()
+                .gt(1000000)
+        ) {
+            return BigNumber.min(
```

Comment:
This is not accurate because it is missing the second half of the liquidity. We should use the common token from the pair to compute the total value locked for a pair in this scenario. In addition, we should consider the double of the common token as the whole liquidity locked in a pool